examples/index.html

```diff
@@ -140,13 +140,15 @@ <h3>Progressive enhancement</h3>
 
       <h3>Custom results</h3>
       <p>
-        This example demonstrates how to allow users to search for a country by either
-        its english name or its endonym. The suggestions are user-defined objects and
+        This example demonstrates how to allow users to search for a country by either:
```

Comment:
Thanks for flagging this bit specifically, @EoinShaughnessy , I think I missed it in my first review.

Unfortunately we can't do the list like this because this file is written in HTML, not Markdown.

Instead, it'll need to look like this:

```
<p>This example demonstrates how to allow users to search for a country by either:</p>
<ul>
    <li>its English name</li>
    <li>the name by which a group of people refer to their country</li>
</ul>
<p>The suggestions are user-defined objects and they are displayed using a user-defined template. Confirming a suggestion will populate the input with the English country name.</p>
```

Reply:
Ahh I see - thanks! That's v useful to know, have noted for future reference. Will add in.